Test/git-issues/git-issue-3978.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN: %testDafnyForEachCompiler "%s"
+
+method Main() returns ()
+{
+ for v_int_7 := 3 to 18
```

Comment:
I see this is how the bug report showed the code. Still, I would suggest renaming `v_int_7` and `v_int_93` to something like `x` and `y`.